### PR TITLE
additional info on devices available to trial/sub

### PR DESCRIPTION
**Pull request by saucechaffe**
I added a section that outlines the differences in devices available to you from a free trial perspective and a subscribed perspective as this appears to be a common ticket and there is no clearly outlined different when signing up or in docs.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/mobile-apps/faq.md [hunk 1]
```diff
@@ -61,6 +61,16 @@ Yes. See [Camera Image Injection](/mobile-apps/features/camera-image-injection).
 For real devices on the public cloud, the limit is 6 hours for Appium tests and 1 hour for Espresso and XCUITest; the same applies to private cloud devices.
 The limits for public cloud devices ensure that there are enough devices available for you, whereas the limits for private cloud devices ensure the longevity of your devices.
 
+#### **Is there a difference in devices I can use as a free trial and with a subscription when live testing?**
```
Comment (aliciakrebs) [inline docs/mobile-apps/faq.md:64]:
```suggestion
#### **What is the difference between the devices available with a free trail and a subscription?**
```

File: docs/mobile-apps/faq.md [hunk 2]
```diff
@@ -61,6 +61,16 @@ Yes. See [Camera Image Injection](/mobile-apps/features/camera-image-injection).
...
+
+For free trial accounts, the only available devices you will see are denoted with a green corner flag that states trial and they will be available to you. If you are subscribed, you will have access to a vast pool of devices which exceeds 300 models and behind those models will be different versions of OS available to you.
```
Comment (aliciakrebs) [inline docs/mobile-apps/faq.md:66]: I assumed it is denoted by a flag that reads "Trail"?  
```suggestion
For free trial accounts, the available devices are denoted with a green Trial flag. If you are subscribed, you have access to a pool of over 300 device models with additional supported OS/version configurations. 
```

File: docs/mobile-apps/faq.md [hunk 3]
```diff
@@ -61,6 +61,16 @@ Yes. See [Camera Image Injection](/mobile-apps/features/camera-image-injection).
 For real devices on the public cloud, the limit is 6 hours for Appium tests and 1 hour for Espresso and XCUITest; the same applies to private cloud devices.
 The limits for public cloud devices ensure that there are enough devices available for you, whereas the limits for private cloud devices ensure the longevity of your devices.
 
+#### **Is there a difference in devices I can use as a free trial and with a subscription when live testing?**
+
+For free trial accounts, the only available devices you will see are denoted with a green corner flag that states trial and they will be available to you. If you are subscribed, you will have access to a vast pool of devices which exceeds 300 models and behind those models will be different versions of OS available to you. 
+
+Example: iPhone 12
```
Comment (aliciakrebs) [inline docs/mobile-apps/faq.md:68]: I am not sure what this example is telling me in regards to the question.

File: docs/mobile-apps/faq.md (hunk 1 above)
Comment (saucechaffe) [reply docs/mobile-apps/faq.md:64]: Please double check typo on trial - here it is trail

File: docs/mobile-apps/faq.md (hunk 3 above)
Comment (saucechaffe) [reply docs/mobile-apps/faq.md:68]: There were a list of OS versions under line 68. Something like: 
Example: iPhone 12
iPhone 12 - iOS 15
iPhone 12 - iOS 16
iPhone 12 - iOS 16.1 
and so on, the example should demonstrate that there isn't only 1 iPhone/device, but multiple running multiple versions of OS

File: docs/mobile-apps/faq.md (hunk 1 above)
Comment (aliciakrebs) [reply docs/mobile-apps/faq.md:64]: fixed, thanks.

File: docs/mobile-apps/faq.md (hunk 3 above)
Comment (aliciakrebs) [reply docs/mobile-apps/faq.md]: @saucechaffe, I updated the example.

File: docs/mobile-apps/faq.md [hunk 4]
```diff
@@ -61,6 +61,16 @@ Yes. See [Camera Image Injection](/mobile-apps/features/camera-image-injection).
...
 The limits for public cloud devices ensure that there are enough devices available for you, whereas the limits for private cloud devices ensure the longevity of your devices.
 
+#### **What is the difference between the devices available with a free trial and a subscription?**
+
+For free trial accounts, the available devices are denoted with a green Trial flag. If you are subscribed, you have access to a pool of over 300 device models with additional supported OS/version configurations. 
+
+For example, with a subscription, you can test iPhone 12 devices with any of these iOS configurations:
+iPhone 12 - iOS 15
+iPhone 12 - iOS 16
+iPhone 12 - iOS 16.1
```
Comment (aoleszyn) [inline docs/mobile-apps/faq.md:71]:
```suggestion
For example, with a subscription, you can test iPhone 12 devices with any of these iOS configurations:
* iPhone 12 - iOS 15
* iPhone 12 - iOS 16
* iPhone 12 - iOS 16.1
```

**Pull request merged**